modules/nw-proxy-configure-object.adoc

```diff
@@ -93,21 +93,27 @@ spec:
   trustedCA:
     name: user-ca-bundle <5>
 ----
++
+--
 <1> A proxy URL to use for creating HTTP connections outside the cluster. The
 URL scheme must be `http`.
 <2> A proxy URL to use for creating HTTPS connections outside the cluster. If
 this is not specified, then `httpProxy` is used for both HTTP and HTTPS
 connections.
 <3> A comma-separated list of destination domain names, domains, IP addresses or
-other network CIDRs to exclude proxying. Preface a domain with `.` to include
-all subdomains of that domain. Use `*` to bypass proxy for all destinations.
-Note that if you scale up workers not included in `networking.machineNetwork[].cidr` from the installation configuration, you must add them to this list to prevent connection issues.
+other network CIDRs to exclude proxying.
++
+Preface a domain with `.` to include all the subdomains of that domain. Use `*` to bypass the proxy for all destinations.
+If you scale up workers that are not included in the network defined by the `networking.machineNetwork[].cidr` field from the installation configuration, you must add them to this list to prevent connection issues.
```

Comment:
Should line 107 be included on 106, to avoid hard wrapping?

This seems to be rendering incorrectly, though I can't put my finger on why it would be doing this:
- Line 106 has an inconsistent line break compared to the break for line 109 in the rendered version; the spacing is off.
- The additional text after the initial sentence for callout three appears to have a different font size.

If there's not an obvious way to fix those rendering issue, it might be better to have the text unbroken, similar to how it was before. Although I agree the line breaks make it easier to read. :thinking:

Reply:
@codyhoag, the formatting is a long standing issue: https://github.com/openshift/openshift-docs/pull/30042

Reply:
ah, nice! With those formatting issues corrected, I don't have any further complaints. :smile: